code/src/components/URLS.js

```diff
@@ -0,0 +1,10 @@
+export const API__KEY = 'daf1a56c904ca6e2943af1246db74e3f'
+export const MOVIELIST__URL = `https://api.themoviedb.org/3/movie/popular?api_key=${API__KEY}&language=en-US&page=1`
+export const TOP_RATED__URL =`https://api.themoviedb.org/3/movie/top_rated?api_key=${API__KEY}&language=en-US&page=1`
```

Comment:
You don't have to differentiate amount of underscores in URL variable name, it's not BEM ;)
For URLs, one underscore between words is enough : `TOP_RATED_URL`